app/javascript/chat/chat.jsx

```diff
@@ -1733,34 +1733,37 @@ export class Chat extends Component {
           ? Object.values(channelUsers[activeChannelId])
               .filter((user) => user.username.match(filterRegx))
               .map((user) => (
-                <div
-                  key={user.username}
-                  className="mention__user"
-                  role="button"
-                  onClick={this.addUserName}
-                  tabIndex="0"
-                  data-content={user.username}
-                  onKeyUp={(e) => {
-                    if (e.keyCode === 13) this.addUserName();
-                  }}
-                >
-                  <img
-                    className="mention__user__image"
-                    src={user.profile_image}
-                    alt={user.name}
-                    style={!user.profile_image ? { display: 'none' } : ' '}
-                  />
-                  <span
-                    style={{
-                      padding: '3px 0px',
-                      'font-size': '16px',
-                    }}
-                  >
-                    {'@'}
-                    {user.username}
-                    <p>{user.name}</p>
-                  </span>
-                </div>
+                <ul className="ulist">
```

Comment:
For the new classNames it would be good to use [BEM](http://getbem.com/) like in other places in the app.

Reply:
If you need help understanding what BEM feel free to ask 🙂